src/components/Register/index.jsx

```diff
@@ -0,0 +1,98 @@
+import { useState } from "react";
+import { useNavigate } from "react-router-dom";
+import { Icon } from "../Icons/style";
+import {
+  Button,
+  Form,
+  Input,
+  InputContainer,
+  RegisterContainer,
+  Title,
+  RegisterStatus,
+  RegisterErrors,
+} from "./style";
+import { register } from "../../requests/auth/register";
+
+export function Register() {
+  const [email, setEmail] = useState("");
+  const [password, setPassword] = useState("");
+  const [name, setName] = useState("");
+  const navigate = useNavigate();
+  const [registrationStatus, setRegistrationStatus] = useState([]);
+
+  const handleSubmit = async (event) => {
+    event.preventDefault();
+
+    const response = await register({ name, email, password });
+    if (response.message === "OK") {
+      setRegistrationStatus({ status: "success" });
+      setTimeout(() => {
+        navigate("/Login");
```

Comment:
Maybe instead of redirecting to login we could "do" the login and redirect to catalog.
Since the login it's now ready and functional it could be just some extra lines.

Reply:
done!